packages/flutter/lib/src/cupertino/text_selection_toolbar.dart

```diff
@@ -187,8 +187,8 @@ class _RenderCupertinoTextSelectionToolbarShape extends RenderShiftedBox {
   _RenderCupertinoTextSelectionToolbarShape(
     this._anchor,
     this._isAbove,
-    RenderBox? child,
-  ) : super(child);
+    super.child,
+  ) ;
```

Comment:
nit: remove this space:

```suggestion
  );
```